lib/helpers.ts

```diff
@@ -91,24 +91,21 @@ export function regexOperations(post: any, keywords: string[]) {
   const termsRegex = new RegExp(`(${keywordWordBoundary.join(")|(")})`, "gi");
 
   const marked: string = mrkdwn(decode(post.text)).text;
+  const combined = combineText(post);
 
-  // We use String.replace here so that we can know which capture group is
-  // actually matched, so that we can extract the appropriate keyword.
-  combineText(post).replace(termsRegex, (_, ...terms: string[]) => {
+  let match;
+  while ((match = termsRegex.exec(combined))) {
```

Comment:
is this an equals sign `=` or an equality `==` comparison?

Reply:
Equals. In this situation, the extra parens are meant to indicate that this is an intentional assignment. See https://eslint.org/docs/latest/rules/no-cond-assign